doc/gemma_online.md

```diff
@@ -0,0 +1,34 @@
+===GEMMA Online API pagina
+Onderstaande tekst zou een basis kunnen vormen voor een inleidende pagina op http://gemmaonline.nl/ over de API standaarden voor de Common ground.
+
+
+Met de <a href="https://vng.nl/samen-organiseren/common-ground">Common Ground</a> en de bijbehorende, andere manier waarop de gemeentelijke ICT-systemen opnieuw georganiseerd worden zijn ook nieuwe uitwisselingsstandaarden nodig. Gegevens worden niet meer en masse gesynchroniseerd maar opgehaald bij de bron op het moment dat ze nodig zijn. De technieken die hiervoor nodig zijn vereisen een andere soort uitwisselingsstandaarden.
```

Comment:
Kom op @michielverhoef URL in markdown formaat :)

```suggestion
Met de [Common Ground](https://vng.nl/samen-organiseren/common-ground) en de bijbehorende, andere manier waarop de gemeentelijke ICT-systemen opnieuw georganiseerd worden zijn ook nieuwe uitwisselingsstandaarden nodig. Gegevens worden niet meer en masse gesynchroniseerd maar opgehaald bij de bron op het moment dat ze nodig zijn. De technieken die hiervoor nodig zijn vereisen een andere soort uitwisselingsstandaarden.
```

Reply:
Aangezien dit een pagina voor GEMMA Online moet worden en ik daar ook geen markdown kan gebruiken tik ik het lekker in HTML in, dat kan ik dromen ;-)